packages/core/src/Stores/traders-hub-store.js

```diff
@@ -387,19 +387,33 @@ export default class TradersHubStore extends BaseStore {
     }
 
     getAvailableMt5Accounts() {
+        const { active_accounts } = this.root_store.client;
+        const result = active_accounts.some(acc => acc.landing_company_shortcode === 'svg');
+
         if (this.is_eu_user && !this.is_demo_low_risk) {
             this.available_mt5_accounts = this.available_cfd_accounts.filter(account =>
                 ['EU', 'All'].some(region => region === account.availability)
             );
             return;
         }
 
+        // Only show financial account for Australian/Norway clients
+        if (!result && this.selected_region === 'Non-EU') {
+            this.available_mt5_accounts = this.available_cfd_accounts.filter(account =>
+                ['Financial'].some(region => region === account.name)
```

Comment:
@aizad-deriv why you create array with just one element ['Financial']? Maybe better will be right like this:
account.name === 'Financial'